homeworks/tatyana.volkorezova_volkorezova/7-class/baseTask.js

```diff
@@ -0,0 +1,43 @@
+const WHEEL_AMOUNT = 4;
+const wheelsPackForVolvo = [];
+
+class Car {
+    constructor(motor, carWheels, kuzovType, sizes, weight) {
+        this.motor = motor;
+        this.carWheels = carWheels;
+        this.kuzovType = kuzovType;
+        this.sizes = sizes;
+        this.weight = weight;
+    }
+}
+
+class Motor {
+    constructor(capacity, energy) {
+        this.capacity = capacity;
+        this.energy = energy;
+    }
+}
+
+class CarWheel {
+    constructor(radius, wheelWidth) {
+        this.radius = radius;
+        this.wheelWidth = wheelWidth;
+    }
+}
+
+class CarSize {
+    constructor(height, width, carLength) {
+        this.height = height;
+        this.width = width;
+        this.carLength = carLength;
+    }
+}
+
+for (let i = 1; i <= WHEEL_AMOUNT; i++) {
+    wheelsPackForVolvo[i] = new CarWheel('16 duyms', '200 mm');
```

Comment:
![image](https://user-images.githubusercontent.com/96523347/213702589-18f44fac-c365-4ea6-9b07-dc6767882d88.png)
![image](https://user-images.githubusercontent.com/96523347/213702899-e60e102a-ea09-4bdc-8149-590de769ebd2.png)

upper case in object key?

Reply:
renamed keys